tlv493d/tlv493d.go

```diff
@@ -403,7 +403,15 @@ func (d *Dev) ReadContinuous(frequency physic.Frequency, precision Precision) (<
 			case <-t.C:
 				value, err := d.Read(precision)
 				if err != nil {
-					// In continuous mode, we'll ignore errors silently.
+					// Try resetting the sensor to recover from errors
+					fmt.Println("Error reading from TLV493D sensor:", err)
```

Comment:
I'm uncomfortable with leaving fmt.Println() in the code. If you really feel it needs to be logged you can send it to log.Printf() but I'd prefix it with the device name so that if there's multiple devices the user can differentiate.

Reply:
Oups, indeed, sorry for that. I kept logs in case of reset as it might indicate an electrical/I2C bus issue.